sandbox.go

```diff
@@ -306,23 +346,25 @@ func (sb *sandbox) populateNetworkResources(ep *endpoint) error {
 
 func (sb *sandbox) clearNetworkResources(ep *endpoint) error {
 
-	for _, i := range sb.osSbox.Info().Interfaces() {
-		// Only remove the interfaces owned by this endpoint from the sandbox.
-		if ep.hasInterface(i.SrcName()) {
-			if err := i.Remove(); err != nil {
-				log.Debugf("Remove interface failed: %v", err)
+	if sb.osSbox != nil {
```

Comment:
Again need to hold the lock while checking for this. So it's best to cache the osSbox while holding the lock and then go through this code snipper with the cached value